development/issue_workflow.md

```diff
@@ -19,6 +19,8 @@ Labels are designated with a diamond shape in the flow diagram.
 
 - **actionable**: Requests that are ready to be handed to development for scheduling.
 
+- **blocker**: This request is preventing another issue within the an sprint from being worked on. Ideally, this should be prioritized above other requests.
```

Comment:
Since we don't use sprints in the agile sense we should likely avoid using it here. I think the following would be fine:

"This request is preventing another issue from being worked on"